ci-operator/config/openshift/installer/openshift-installer-release-4.18.yaml

```diff
@@ -475,6 +475,16 @@ tests:
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-gcp
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-gcp-user-provisioned-dns
+  optional: true
+  steps:
+    cluster_profile: gcp
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      USER_PROVISIONED_DNS: "yes"
+    workflow: openshift-e2e-gcp
```

Comment:
This is using the existing e2e workflow, but I assumed we would need a new workflow in which we (users) create the public DNS record. This is what we expect users to do and I suspect will be necessary for e2e tests to be able to run.

Reply:
I am not sure what the issues the e2e tests would encounter. Once the install has completed, there would be a CoreDNS pod running on the control plane that can provided DNS resolution for API and API-Int. I am not sure if the e2e tests need anything more than that. 

At this time using the test to make sure install is successful would be the 1st step. We can evaluate additional setup necessary for e2e tests to pass after we reach that milestone.

Reply:
The problem is that when the e2e test pod tries to connect to the cluster (via kubeconfig), it will not be able to resolve the dns for the api. The coreDNS pods can only provide dns for the cluster itself.